config/jobs/kubernetes/kops/build_vars.py

```diff
@@ -31,7 +31,13 @@
 
 # GCE distributions
 gce_distro_options = [
+    "deb12",
+    "deb12arm64",
     "u2204",
+    "u2404",
+    "u2404arm64",
+    "umini2404",
+    "umini2404arm64",
```

Comment:
why mini?

Reply:
testing `ubuntu-minimal-2404` https://gcloud-compute.com/images.html which as less system packages IIUC.

Reply:
otherwise LGTM, once the tests pass.